File: CS/WindowsApplication1/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraPrinting.Preview;
using DevExpress.XtraReports.UI;

namespace WindowsApplication1 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button2_Click(object sender, EventArgs e) {
            ReportPrintTool printTool = new ReportPrintTool(new XtraReport1());
            printTool.PreviewForm.VisibleChanged += new EventHandler(PreviewFormEx_VisibleChanged);
            printTool.ShowPreviewDialog(); 
        }

        void PreviewFormEx_VisibleChanged(object sender, EventArgs e) {
            ((PrintPreviewFormEx)sender).PrintBarManager.AllowQuickCustomization = false;
            ((PrintPreviewFormEx)sender).PrintBarManager.Bars[2].OptionsBar.DrawDragBorder = false;
            ((PrintPreviewFormEx)sender).PrintBarManager.Bars[0].OptionsBar.DrawDragBorder = false;
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button2.Click
			Dim r As New XtraReport1()
			AddHandler r.PrintingSystem.PreviewFormEx.VisibleChanged, AddressOf PreviewFormEx_VisibleChanged
			r.ShowPreviewDialog()

		End Sub

		Private Sub PreviewFormEx_VisibleChanged(ByVal sender As Object, ByVal e As EventArgs)
			CType(sender, DevExpress.XtraPrinting.Preview.PrintPreviewFormEx).PrintBarManager.AllowQuickCustomization = False
			CType(sender, DevExpress.XtraPrinting.Preview.PrintPreviewFormEx).PrintBarManager.Bars(2).OptionsBar.DrawDragBorder = False
			CType(sender, DevExpress.XtraPrinting.Preview.PrintPreviewFormEx).PrintBarManager.Bars(0).OptionsBar.DrawDragBorder = False
		End Sub
	End Class
End Namespace